python/SDtesting.py

```diff
@@ -0,0 +1,66 @@
+#######################################################
+# PIP INSTALLS TO MAKE FOR NLTK LDA TOPIC MODELING
+# pip install gensim
+# pip install pyldavis
+# pip install nltk
+# pip install ipython
+# pip install regex
+# ####################################################
+import pandas as pd
+import nltk
+nltk.download('stopwords')
+from nltk.corpus import stopwords
+import string
+import gensim.corpora as corpora
+from gensim.models import LdaModel
+import pyLDAvis.gensim_models
+import os
+import re
+
+#Variable with current dir
+workingDir = os.getcwd()
```

Comment:
should we get rid of workingDir and parentDir? They're not used anywhere else and we can use strings to define stardewPath